include/deepworks/shape.hpp

```diff
@@ -1,9 +1,19 @@
 #pragma once
 
 #include <vector>
+#include <ostream>
 
 namespace deepworks {
 
 using Shape = std::vector<int>;
 
+inline std::ostream& operator<<(std::ostream& os, Shape shape) {
+    os << "[";
+    for (auto sh : shape) {
+        os << sh << " ";
```

Comment:
Let's add delimiter: 
```suggestion
        os << sh << ," ";
```

Reply:
So, then it will be `[, 3, 5, 5]`

Reply:
BTW comma outside of string :(

Reply:
Fixed